apps/captiveportal/main.cpp

```diff
@@ -77,11 +77,11 @@ Q_DECL_EXPORT int main(int argc, char *argv[])
 
     QString translationPath("/usr/share/translations/");
     QTranslator engineeringEnglish;
-    engineeringEnglish.load("sailfish-browser_eng_en", translationPath);
+    engineeringEnglish.load("sailfish-captiveportal_eng_en", translationPath);
     qApp->installTranslator(&engineeringEnglish);
 
     QTranslator translator;
-    translator.load(QLocale(), "sailfish-browser", "-", translationPath);
+    translator.load(QLocale(), "sailfish-captiveportal", "-", translationPath);
     qApp->installTranslator(&translator);
 
     //% "Browser"
```

Comment:
```suggestion
    //% "Captive Portal"
    view->setTitle(qtTrId("sailfish-captiveportal-ap-name"));
```
+ updating sailfish-captiveportal.desktop accordingly.

Reply:
Regarding translation itself if you have suggestions, I'm all ears. It could indicate more towards WLAN login.

Reply:
Well,
-  WLAN portal,
-  Login portal, or
- Network login portal

even perhaps. That "login portal" could be even "login page" I think but maybe not in this context (app title).